Cargo.toml

```diff
@@ -66,6 +66,7 @@ incremental = false
 codegen-units = 16
 
 [patch.crates-io]
+holochain_wasmer_host = {git = "https://github.com/holochain/holochain-wasmer.git", branch = "refactor/reinstate-module-cache"}
```

Comment:
The CI won't accept this unfortunately

Reply:
Ok, I just wanted to run the tests while there's no wasmer crate. Why doesn't that work?